### DISCUSSION TITLE
Hide abbreviated abbreviation

### DISCUSSION THREAD

**Discussion by WilliamWelsh** [fish-shell/fish-shell#9330] [Q&A] [answered]
Hi, is there a way to hide an abbreviation so when I type it, it doesn't change to the abbreviation?
For example, I have `ll` do `colorls --sd -1`, can I make it so if I type `ll` it says as `ll` visually and still acts like `colorls --sd -1`?

**Comment by UlisesAlexanderAM** [comment, marked as answer]
I think you'd prefer to use an alias rather than an abbreviation. For the one you're using, you can type this:
`alias -s ll "colorls --sd -1"`

**Reply by WilliamWelsh** [reply]
Works like a charm. Thanks a bunch.

**Reply by UlisesAlexanderAM** [reply]
Glad to help.
